data/json/items/tool/fishing.json

```diff
@@ -145,14 +150,16 @@
     "to_hit": { "grip": "solid", "length": "long", "surface": "any", "balance": "neutral" },
     "qualities": [ [ "FISHING_ROD", 2 ] ],
     "extend": { "flags": [ "FRAGILE_MELEE" ] },
+    "//": "Unidentified entry above named ,extend, leaving here to avoid errors until further clarification on its function",
```

Comment:
?? What do you mean by this? What's unidentified? Do you not understand what the ``extend`` field does?

You really should just... read the documentation. Or ask. ``extend`` is used to add onto fields that support arrays when the object uses ``copy-from``, so that flags don't have to be redefined. Which you just redefined them. You're doing it backwards here.